src/js/helpers/LoadHelpers.js

```diff
@@ -438,7 +438,7 @@ export function getUSFMParams(projectPath, manifest) {
     else if (manifest.ts_project) bookAbbr = manifest.ts_project.id;
     let direction = manifest.target_language.direction;
     let params = {
-        originalLanguagePath: ORIGINAL_LANGUAGE_PATH,
+        originalLanguagePath: '',
```

Comment:
The params variable is being removed and ORIGINAL_LANGUAGE_PATH was undefined anyway.